charts/blobscan/values.yaml

```diff
@@ -24,16 +24,19 @@ httpPort: 3000
 # -- Command arguments
 args: []
 
+# -- Application configuration
+config:
+  NEXTAUTH_URL: ""
+  DATABASE_URL: "postgresql://postgres:postgres@blobscan-blobscandb:5432/blobscan?schema=public"
+  SECRET_KEY: "supersecret"
+  NEXT_PUBLIC_NETWORK_NAME: "mainnet"
+  NEXT_PUBLIC_BEACON_BASE_URL: ""
+  NEXT_PUBLIC_EXPLORER_BASE_URL: ""
```

Comment:
Let's keep the sane defaults so it works out of the box until we configure these variables as optional

Reply:
Set them to sane defaults.